Client/src/Components/TerryComponent/reusable/AlertMessage.jsx

```diff
@@ -0,0 +1,74 @@
+// eslint-disable-next-line no-unused-vars
+import React from "react";
+import "./AlertMessage.css";
+import PropTypes from "prop-types";
+import { Snackbar, Alert } from "@mui/material";
+
+const AlertMessage = ({
+  open,
+  type,
+  title = "",
+  message,
+  dismissText = "Learn more",
+  actionText,
+  close,
+  dismissClose,
+}) => {
+  // Function to determine icon color based on the title
+  const switchIconColor = (type) => {
```

Comment:
Functions should al linclude JSDocs so that their usage is well documented and so that we benefit from itellisense suggestion and autocomplete features.

Please see other components and functions for implementations if you've not worked iwth JSdocs before